PHP/hw4.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$n = 20;
+$ans = "";
+
+function judgePrime($m){
+    if ($m==1) return true;
+
+    for ($i=2; $i<=$m; $i++) if ($m%$i==0) break;
+    
+    if ($m==$i) return true;
+    else return false;
+}
+
+for ($i=1; $i<=$n; $i++) if (judgePrime($i)) $ans .= " $i";
```

Comment:
The homework: `implement a function to return the prime numbers between 1 and n`, therefore you need one more function and return an array of prime number. Example.
```
function getPrimeNumbers($n) {
  // Loop from 1 to n 
  // Check if number is prime.
  //Append to an array
  // Return the array
}

echo getPrimeNumbers(100);

Reply:
Please also fix the source code following PSR-2 standard.